Tasks/XcodeV5/xcode.ts

```diff
@@ -21,6 +21,7 @@ async function run() {
         if (xcodeVersionSelection === 'specifyPath') {
             let devDir = tl.getInput('xcodeDeveloperDir', true);
             tl.setVariable('DEVELOPER_DIR', devDir);
+            telemetryData.xcodeDeveloperDir = devDir;
```

Comment:
This seem to be some user data which we don't need to gather for telemetry

Reply:
@anatolybolshakov , I don't think so.
The most of the users use this option to select Xcode.
The task only provides ability to select Xcode 9, 10, 11, 12. Without specifying exact version.
If user want to select Xcode 11.3, he has to select "specifyPath" option and put path from docs: https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md#xcode

Reply:
Yep, but it could include some personal data in path - could it make sense to maybe just gather usage of this (was it specified or not)? Or maybe think about ways to extract only version info

Reply:
Fixed